packages/components/src/internal/util/utils.test.ts

```diff
@@ -1417,3 +1417,46 @@ describe('getValueFromRow', () => {
         expect(getValueFromRow(row, 'bogus')).toEqual(undefined);
     });
 });
+
+describe('makeCommaSeparatedString', () => {
+    test('values is empty', () => {
+        expect(makeCommaSeparatedString(undefined)).toBe('');
+        expect(makeCommaSeparatedString(null)).toBe('');
+        expect(makeCommaSeparatedString([])).toBe('');
+    });
+
+    test('values has single item', () => {
+        expect(makeCommaSeparatedString(['blood'])).toBe('blood');
+        expect(makeCommaSeparatedString([123])).toBe('123');
+        expect(makeCommaSeparatedString([true])).toBe('true');
+    });
+
+    test('values > 1', () => {
+        expect(makeCommaSeparatedString(['blood', 'dna'])).toBe('blood and dna');
+        expect(makeCommaSeparatedString(['blood', 'dna', 'plasma'])).toBe('blood, dna and plasma');
+        expect(makeCommaSeparatedString([123, 456])).toBe('123 and 456');
+        expect(makeCommaSeparatedString([123, 'blood', true])).toBe('123, blood and true');
+    });
+});
+
+describe('getValuesSummary', () => {
+    test('values is empty', () => {
+        expect(getValuesSummary(undefined, 'sample')).toBe('');
+        expect(getValuesSummary(null, 'sample')).toBe('');
+        expect(getValuesSummary([], 'sample')).toBe('');
+    });
+
+    test('values has single item', () => {
+        expect(getValuesSummary(['blood'], 'sample')).toBe('1 sample (blood)');
+        expect(getValuesSummary(['blood'], 'sample', 'samplePlural')).toBe('1 sample (blood)');
```

Comment:
I don't see a test case where the custom plural value is used.

Reply:
It's used in the test case below.`expect(getValuesSummary(['blood', 'dna', 'plasma'], 'sample', 'samplePlural')).toBe('3 samplePlural (blood, dna and plasma)');`